src/OpenTK.NT/Native/User32/Structs/Msg.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Runtime.InteropServices;
+
+using HWND = System.IntPtr;
+using LPARAM = System.IntPtr;
+using WPARAM = System.IntPtr;
+
+namespace OpenTK.NT.Native
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct Msg
```

Comment:
Is this the native name?

Reply:
Yes - I wasn't sure whether I should call it `Message`. If I do, the field `Message` would have to be renamed though. Or the struct would have to get a different name. Generally not sure on this.